src/view/com/util/List.web.tsx

```diff
@@ -230,30 +243,119 @@ function useResizeObserver(
 }
 
 let Row = function RowImpl<ItemT>({
+  itemKey,
   item,
   index,
   renderItem,
   extraData: _unused,
+  onViewableItemsChanged,
 }: {
+  itemKey: string
   item: ItemT
   index: number
   renderItem:
     | null
     | undefined
     | ((data: {index: number; item: any; separators: any}) => React.ReactNode)
   extraData: any
+  onViewableItemsChanged: FlatListProps<ItemT>['onViewableItemsChanged']
 }): React.ReactNode {
   if (!renderItem) {
     return null
   }
   return (
     <View style={styles.row}>
-      {renderItem({item, index, separators: null as any})}
+      <RowVisibility
+        itemKey={itemKey}
+        item={item}
+        index={index}
+        onViewableItemsChanged={onViewableItemsChanged}>
+        {renderItem({item, index, separators: null as any})}
+      </RowVisibility>
     </View>
   )
 }
 Row = React.memo(Row)
 
+let RowVisibility = function <ItemT>({
+  itemKey,
+  item,
+  index,
+  onViewableItemsChanged,
+  children,
+}: {
+  itemKey: string
+  item: ItemT
+  index: number
+  onViewableItemsChanged: FlatListProps<ItemT>['onViewableItemsChanged']
+  children: React.ReactNode
+}): React.ReactNode {
+  if (!onViewableItemsChanged) {
```

Comment:
Potential hazard if this is supplied conditionally. Might be better to just always wrap.